File: VehicleTransport/主版本/3.开发源代码/客户端/手机客户端/Last of KJ915_bestway/src/com/bestway/kj915/domain/reflect/m_Person.java
package com.bestway.kj915.domain.reflect;

public class m_Person {

	public int ID;
	public String vc_Code;// 工号
	public String vc_Name;// 姓名
	public int i_Sex;// 性别
	public int DepartmentID;// 部门编号
	public String vc_Job;// 职务
	public String vc_WorkType;// 工种
	public String vc_Telphone;// 电话
	public String vc_Memo;// 备注
	public int i_Flag = 0;
	
	public int getID() {
		return ID;
	}
	public void setID(int iD) {
		ID = iD;
	}
	public String getVc_Code() {
		return vc_Code;
	}
	public void setVc_Code(String vc_Code) {
		this.vc_Code = vc_Code;
	}
	public String getVc_Name() {
		return vc_Name;
	}
	public void setVc_Name(String vc_Name) {
		this.vc_Name = vc_Name;
	}
	public int getI_Sex() {
		return i_Sex;
	}
	public void setI_Sex(int i_Sex) {
		this.i_Sex = i_Sex;
	}
	public int getDepartmentID() {
		return DepartmentID;
	}
	public void setDepartmentID(int departmentID) {
		DepartmentID = departmentID;
	}
	public String getVc_Job() {
		return vc_Job;
	}
	public void setVc_Job(String vc_Job) {
		this.vc_Job = vc_Job;
	}
	public String getVc_WorkType() {
		return vc_WorkType;
	}
	public void setVc_WorkType(String vc_WorkType) {
		this.vc_WorkType = vc_WorkType;
	}
	public String getVc_Telphone() {
		return vc_Telphone;
	}
	public void setVc_Telphone(String vc_Telphone) {
		this.vc_Telphone = vc_Telphone;
	}
	public String getVc_Memo() {
		return vc_Memo;
	}
	public void setVc_Memo(String vc_Memo) {
		this.vc_Memo = vc_Memo;
	}
	public int getI_Flag() {
		return i_Flag;
	}
	public void setI_Flag(int i_Flag) {
		this.i_Flag = i_Flag;
	}
	
	
	public m_Person() {
		super();
	}
	public m_Person(int iD, String vc_Code, String vc_Name, int i_Sex,
			int departmentID, String vc_Job, String vc_WorkType,
			String vc_Telphone, String vc_Memo, int i_Flag) {
		super();
		ID = iD;
		this.vc_Code = vc_Code;
		this.vc_Name = vc_Name;
		this.i_Sex = i_Sex;
		DepartmentID = departmentID;
		this.vc_Job = vc_Job;
		this.vc_WorkType = vc_WorkType;
		this.vc_Telphone = vc_Telphone;
		this.vc_Memo = vc_Memo;
		this.i_Flag = i_Flag;
	}
	@Override
	public String toString() {
		return "m_Person [ID=" + ID + ", vc_Code=" + vc_Code + ", vc_Name="
				+ vc_Name + ", i_Sex=" + i_Sex + ", DepartmentID="
				+ DepartmentID + ", vc_Job=" + vc_Job + ", vc_WorkType="
				+ vc_WorkType + ", vc_Telphone=" + vc_Telphone + ", vc_Memo="
				+ vc_Memo + ", i_Flag=" + i_Flag + "]";
	}
	

}

File: VehicleTransport/主版本/3.开发源代码/DB_VehicleTransportManage/Model/m_Person.cs
/**  版本信息模板在安装目录下，可自行修改。
* m_Person.cs
*
* 功 能： N/A
* 类 名： m_Person
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2014-08-11 11:08:33   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace DB_VehicleTransportManage.Model
{
	/// <summary>
	/// m_Person:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class m_Person
	{
		public m_Person()
		{}
		#region Model
		private int _id;
		private string _vc_code;
		private string _vc_name;
		private int? _i_sex;
		private int _departmentid;
		private string _vc_job;
		private string _vc_worktype;
		private string _vc_telphone;
		private string _vc_memo;
		private int? _i_flag=0;
		/// <summary>
		/// 
		/// </summary>
		public int ID
		{
			set{ _id=value;}
			get{return _id;}
		}
		/// <summary>
		/// 工号
		/// </summary>
		public string vc_Code
		{
			set{ _vc_code=value;}
			get{return _vc_code;}
		}
		/// <summary>
		/// 姓名
		/// </summary>
		public string vc_Name
		{
			set{ _vc_name=value;}
			get{return _vc_name;}
		}
		/// <summary>
		/// 性别
		/// </summary>
		public int? i_Sex
		{
			set{ _i_sex=value;}
			get{return _i_sex;}
		}
		/// <summary>
		/// 部门编号
		/// </summary>
		public int DepartmentID
		{
			set{ _departmentid=value;}
			get{return _departmentid;}
		}
		/// <summary>
		/// 职务
		/// </summary>
		public string vc_Job
		{
			set{ _vc_job=value;}
			get{return _vc_job;}
		}
		/// <summary>
		/// 工种
		/// </summary>
		public string vc_WorkType
		{
			set{ _vc_worktype=value;}
			get{return _vc_worktype;}
		}
		/// <summary>
		/// 电话
		/// </summary>
		public string vc_Telphone
		{
			set{ _vc_telphone=value;}
			get{return _vc_telphone;}
		}
		/// <summary>
		/// 备注
		/// </summary>
		public string vc_Memo
		{
			set{ _vc_memo=value;}
			get{return _vc_memo;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? i_Flag
		{
			set{ _i_flag=value;}
			get{return _i_flag;}
		}
		#endregion Model

	}
}

